tests/unit/responses/crossref/2018.3804742.yml

```diff
@@ -0,0 +1,188 @@
+---
```

Comment:
can you put all the crossref responses literally as we received them? so in `json` format. `expected` is fine in `yaml`.